enos/enos-modules.hcl

```diff
@@ -67,7 +67,6 @@ module "vault_verify_agent_output" {
 
 module "vault_cluster" {
   source = "app.terraform.io/hashicorp-qti/aws-vault/enos"
-  # source = "../../terraform-enos-aws-vault"
```

Comment:
😭, I leave there here for easy debugging but we could always declare a `vault_cluster_local` module and then swap the scenario we're working on to use it.

Reply:
Whoops, I had been messing with this to debug some provider stuff and probably inadvertently assumed this line was one of my messings-with. I'll put it back for now until we have a better solution.